lighthouse-cli/test/fixtures/dobetterweb/dbw_disabled.css

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+body {
+  display: box; /* PASS: in disabled stylesheet */
```

Comment:
This is a little odd, yeah? 

AFAIUI, `display:box` should trigger a fail on the flexbox test if this stylesheet is actually read. So you're asserting that `disabled` works correctly with that.  But the assertion isn't really about the link-blocking-first-paint audit (because it can't be).

If what I described is mostly correct, could you add a comment summarizing the approach in there?